packages/cli/test/simulation/fourNodeSingleValidator.test.ts

```diff
@@ -0,0 +1,32 @@
+import {nodeAssertions} from "../utils/simulation/assertions.js";
+import {SimulationEnvironment} from "../utils/simulation/index.js";
+
+// TODO: Enable these back after fixing the issue with peer connection and `Error: PublishError.Duplicate`
+describe.skip("Run four nodes, single validator per node, 32 interop validators (no eth1)", function () {
+  this.timeout("5m");
+
+  const env = new SimulationEnvironment({
+    beaconNodes: 4,
+    validatorClients: 1,
+    validatorsPerClient: 32,
+    // We need to use the attesting participation so have to switch to altair
+    altairEpoch: 1,
+    // Use a larger value instead of Infinity
+    // https://github.com/ChainSafe/lodestar/issues/4505
+    bellatrixEpoch: 10 ** 12,
+  });
+
+  before(async function () {
+    await env.start();
+    await env.network.connectAllNodes();
```

Comment:
Should this be done on all sim tests?

Reply:
Yes it should, but we don't run multiple nodes in all tests (for now). Also there is an issue with multiple nodes causing duplication publication error need to debug further. 

This PR is focused only for these assertions: 

- Missed Blocks
- Participation on Head
- Participation on FFG

Rest will come in upcoming PRs and will be improved further as well.